pkg/apis/v1beta1/nodepool_budgets_test.go

```diff
@@ -72,56 +72,48 @@ var _ = Describe("Budgets", func() {
 	})
 	Context("NodePool/AllowedDisruptions", func() {
 		It("should return the min allowedDisruptions", func() {
-			min, err := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
-			Expect(err).To(Succeed())
+			min := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
 			Expect(min).To(BeNumerically("==", 10))
 		})
 		It("should return the min allowedDisruptions, ignoring inactive crons", func() {
 			// Make the first and third budgets inactive
 			budgets[0].Schedule = lo.ToPtr("@yearly")
 			budgets[2].Schedule = lo.ToPtr("@yearly")
-			min, err := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
-			Expect(err).To(Succeed())
+			min := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
 			Expect(min).To(BeNumerically("==", 100))
 		})
 		It("should return MaxInt32 if all crons are inactive", func() {
 			budgets[0].Schedule = lo.ToPtr("@yearly")
 			budgets[1].Schedule = lo.ToPtr("@yearly")
 			budgets[2].Schedule = lo.ToPtr("@yearly")
 			budgets[3].Schedule = lo.ToPtr("@yearly")
-			min, err := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
-			Expect(err).To(Succeed())
+			min := nodePool.GetAllowedDisruptions(ctx, fakeClock, 100)
 			Expect(min).To(BeNumerically("==", math.MaxInt32))
 		})
-		It("should fail and return zero values if a schedule is invalid", func() {
+		It("should return zero values if a schedule is invalid", func() {
```

Comment:
Looks like we are testing the same test case twice?

Reply:
Should we also have a check when the value is invalid, not just the schedule?

Reply:
one is the budget test, and one is the nodepool test. It's effectively the same, so I can go either way on including it.